src/test/stubs.ts

```diff
@@ -0,0 +1,42 @@
+import { uniqueId } from 'lodash'
```

Comment:
This file looks like it's not being used

Reply:
Ah, oops, I'm actually not ready with that part :).